### PR TITLE
Update CONTRIBUTING.md to discourage editing compiled files

### PR DESCRIPTION
**Pull request by NonameSLdev**
Just added another section at the beginning of the page, to let newcomers know what files to edit clearly.

### CODE REVIEW

**Comment by DanielRuf** [comment]
It would be much better to not have any compiled files in the repository like @tomscholz said.

**Comment by tomscholz** [comment]
I think you should add that nobody should commit compiled files aswell.

**Comment by DanielRuf** [comment]
> I think you should add that nobody should commit compiled files aswell.

edit ===  commit here on GitHub if you mean this.

**Comment by tomscholz** [comment]
yes ^^

**Comment by DanielRuf** [comment]
@Dogfalo you can also make changes on PRs by default ;-)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,4 +1,8 @@
 ## Contributing
+- Editing files
+  - **Do not edit compiled files!** If you wish to modify the css edit the `.scss` files.
+  - Don't try to contribute to files in the /dist/ folder or the /bin/ folder.
+  - To edit the materialize framework edit /sass/ and /js/, to edit the materialize website edit the /jade/ folder.
```
Comment (DanielRuf) [inline CONTRIBUTING.md]: @Dogfalo you can also make changes on PRs by default ;-)

**Comment by NonameSLdev** [comment]
@Dogfalo I removed it :)
@tomscholz It says not to edit compiled files... What exactly do you mean?

> I think you should add that nobody should commit compiled files aswell.

**Pull request closed without merging**